packages/vue-component-library/src/components/RnTransfer/README.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Just noticed that you do have docs here, but why do we have docs here And the document folder?